### ISSUE TITLE
AttributeError: module 'html.parser' has no attribute 'HTMLParseError'

### ISSUE THREAD

**Issue by yriveiro** [jakubroztocil/cointrol#9]
I'm following the install instructions and I'm stuck in this error:

```python
(virtualenv) ~/Cointrol ❯❯❯ cointrol/manage.py migrate
Traceback (most recent call last):
  File "cointrol/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/core/management/__init__.py", line 345, in execute
    settings.INSTALLED_APPS
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/conf/__init__.py", line 46, in __getattr__
    self._setup(name)
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/conf/__init__.py", line 42, in _setup
    self._wrapped = Settings(settings_module)
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/conf/__init__.py", line 94, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 978, in _gcd_import
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 936, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 978, in _gcd_import
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 655, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
  File "/Users/yriveiro/Cointrol/cointrol/cointrol/__init__.py", line 8, in <module>
    django.setup()
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/__init__.py", line 18, in setup
    from django.utils.log import configure_logging
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/utils/log.py", line 13, in <module>
    from django.views.debug import ExceptionReporter, get_exception_reporter_filter
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/views/debug.py", line 10, in <module>
    from django.http import (HttpResponse, HttpResponseServerError,
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/http/__init__.py", line 4, in <module>
    from django.http.response import (
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/http/response.py", line 13, in <module>
    from django.core.serializers.json import DjangoJSONEncoder
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/core/serializers/__init__.py", line 23, in <module>
    from django.core.serializers.base import SerializerDoesNotExist
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/core/serializers/base.py", line 6, in <module>
    from django.db import models
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/db/models/__init__.py", line 6, in <module>
    from django.db.models.query import Q, QuerySet, Prefetch  # NOQA
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/db/models/query.py", line 13, in <module>
    from django.db.models.fields import AutoField, Empty
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/db/models/fields/__init__.py", line 18, in <module>
    from django import forms
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/forms/__init__.py", line 6, in <module>
    from django.forms.fields import *  # NOQA
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/forms/fields.py", line 18, in <module>
    from django.forms.utils import from_current_timezone, to_current_timezone
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/forms/utils.py", line 15, in <module>
    from django.utils.html import format_html, format_html_join, escape
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/utils/html.py", line 16, in <module>
    from .html_parser import HTMLParser, HTMLParseError
  File "/Users/yriveiro/Cointrol/virtualenv/lib/python3.6/site-packages/django/utils/html_parser.py", line 12, in <module>
    HTMLParseError = _html_parser.HTMLParseError
AttributeError: module 'html.parser' has no attribute 'HTMLParseError'
```

**Comment by sherifkandeel** [comment]
https://stackoverflow.com/questions/34827566/attributeerror-module-html-parser-has-no-attribute-htmlparseerror

**Comment by yriveiro** [comment]
Thanks for the hint @sherifkandeel

**Comment by Granitosaurus** [comment]
Shouldn't requirements.txt have django upgraded to 1.8 at least then?

**Comment by jakubroztocil** [comment]
Fixed in 50211d787a1014269797a742811a8e1eccefa361
